packages/nouns-contracts/contracts/client-incentives/Rewards.sol

```diff
@@ -320,11 +325,18 @@ contract Rewards is
         require(lastProposalId >= t.nextProposalIdToReward, 'bad lastProposalId');
         require(isSortedAndNoDuplicates(votingClientIds), 'must be sorted & unique');
 
-        NounsDAOTypes.ProposalForRewards[] memory proposals = nounsDAO.proposalDataForRewards(
-            t.nextProposalIdToReward,
-            lastProposalId,
-            votingClientIds
-        );
+        uint16 proposalEligibilityQuorumBps_ = $.params.proposalEligibilityQuorumBps;
+
+        NounsDAOTypes.ProposalForRewards[] memory proposals = nounsDAO.proposalDataForRewards({
+            firstProposalId: t.nextProposalIdToReward,
+            lastProposalId: lastProposalId,
+            proposalEligibilityQuorumBps: proposalEligibilityQuorumBps_,
+            excludeCanceled: true,
+            requireVotingEnded: true,
+            votingClientIds: votingClientIds
+        });
+        require(proposals.length > 0, 'at least one eligible proposal');
+        t.numEligibleProposals = proposals.length;
```

Comment:
perhaps it's best now to remove `t.numEligibleProposals` and instead just use `proposals.length`?
it feels confusing to have this extra var, it makes me assume there's a difference between the proposals list and which are considered eligible, when in fact now they are all eligible.

wdyt?

Reply:
fixed 1ebd2c4666716613632fc1a104ebfbbbbafdbe05